tests/test_boto3_store.py

```diff
@@ -21,7 +21,7 @@ def credentials(request):
     return request.param
 
 
-@pytest.yield_fixture()
+@pytest.fixture
```

Comment:
out of curiosity: what's the difference between `fixture` and `yield_fixture`, and why did you have to change it here?

Reply:
It's just [outdated](https://docs.pytest.org/en/7.1.x/yieldfixture.html), there was a warning